tests/queries/0_stateless/02020_exponential_smoothing.reference

```diff
@@ -654,23 +654,23 @@ exponentialTimeDecayedAvg
 0	48	0.201	████████████████████
 0	49	0.196	███████████████████▌
 Check `exponentialTimeDecayed.*` supports sliding windows
-2	1	3.010050167084	2	3.030251507111	0.993333444442
-1	2	7.060905027605	4.080805360107	4.02030134086	1.756312382816
```

Comment:
I calculated first two values and old refence looks correct:
```python
In [25]: np.exp(0/100) * 2 + np.exp(1/100) * 1 + np.exp(2/100) * 0
Out[25]: 3.010050167084168

In [26]: np.exp(-1/100) * 2 + np.exp(0/100) * 1 + np.exp(1/100) * 0 + np.exp(2/100) * 4
Out[26]: 7.06090502760536
```

May you please check it? I calculated it with the formula $$e ^ {(t_{n-i} - t_{n}) / 100} * v_{n}$$ where `n` is current row, `t` and `v` is time and value, and `i` is for sliding window (`[-2, -1, 0, 1, 2]` for `Rows BETWEEN 2 PRECEDING AND 2 FOLLOWING`)

Reply:
Yes, I did check it, this is actually what was changed:

```
>>> np.exp(-2/100) * 2 + np.exp(-1/100) * 1 + np.exp(0/100) * 0
2.9504471803626786
>>> np.exp(-3/100) * 2 + np.exp(-2/100) * 1 + np.exp(-1/100) * 0 + np.exp(0/100) * 4
6.9210897404037715
```

It is done in order to decay value up to latest value in the window: $\sum\limits_{i=0}^{W-1}  v_{n-i} \cdot e ^ {(t_{n-i} - t_{n}) / 100}$, where $W$ is window size. Otherwise, it would make no sense when the end of window is not a current row.

Reply:
Now I get, thanks